documentation/attributes.md

```diff
@@ -0,0 +1,34 @@
+### Attributes
```

Comment:
Any reason to break attributes into its own nested doc? To me it feels like something that should be on the main README for immediate reference

Reply:
No strong opinion here. Happy to move it to the main README.